core/build.gradle

```diff
@@ -16,6 +16,7 @@
 
 plugins {
     id 'java'
+    id 'com.github.sherter.google-java-format' version '0.9'
```

Comment:
I believe this should just be moved to the root `build.gradle` instead?

Reply:
Addressed in https://github.com/MobilityData/gtfs-validator/pull/953/commits/13bda0f82c0d9d6dbb9290d554d615b29c26150e.